api_spec.yaml

```diff
@@ -1752,6 +1760,81 @@ paths:
                                 type: string
                                 enum:
                                     - Unauthorised access
+    /dashboard/api/tenants/third-party/boxy-saml:
+        put:
+            tags:
+                - Multitenancy
+            summary: Create or update third party config for a tenant with BoxyHQ
+            parameters:
+                - name: authorization
+                  in: header
+                  required: true
+                  schema:
+                      type: string
+                      example: "Bearer API_KEY"
+            requestBody:
+                content:
+                    application/json:
+                        schema:
+                            type: object
+                            properties:
+                                thirdPartyIdSuffix:
+                                    type: string
+                                boxyUrl:
+                                    type: string
```

Comment:
Which of these options take the SAML xml file input ? should it be encoded to base64 or something ? Include all those info in the description. Also probably add link to ours or Boxy documentation about what this API might do

Reply:
Its `encodedRawMetadata`, have also added descriptions to various fields and linked documentation in the body description